### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by work-weicheng**
Tell me whether it's ok or not?

### CODE REVIEW

**Comment by weicheng2138** [comment]
It should be chapter 3!!!

**Comment by work-weicheng** [comment]
> It should be chapter 3!!!

But my mom tell me that it should be 1?
Please check again.

File: README.md [hunk 1]
```diff
@@ -2,3 +2,7 @@
 Test for Pull Request
 
 What you did is going to rain fall the day...
+It seems that you have agree with the previous update
+how about this title?
+
+/* Chapter 1 My computer is going to be wonderful? */
```
Comment (weicheng2138) [inline README.md]: Chapter 1.1 please. Don't do 3
```suggestion
/* Chapter 1.1 My computer is going to be wonderful? */
```

**Review by weicheng2138 — COMMENTED** [review]
review text

**Pull request merged**